patches/0053-Wifi6-Led-Label-Change.patch

```diff
@@ -0,0 +1,195 @@
+++			label = "wf188:green:wifi2g";
++ 			gpios = <&tlmm 24 GPIO_ACTIVE_HIGH>;
++-			linux,default-trigger = "wf188:green:2g";
+++			linux,default-trigger = "wf188:green:wifi2g";
++ 			default-state = "off";
++ 		};
++ 		led@18 {
++-			label = "led_eth";
+++			label = "wf188:green:eth";
++ 			gpios = <&tlmm 18 GPIO_ACTIVE_HIGH>;
++ 			linux,default-trigger = "wf188:green:eth";
++ 			default-state = "off";
++ 		};
++                 led_power: led@16 {
++-                        label = "led_pwr";
+++                        label = "wf188:green:power";
++                         gpios = <&tlmm 16 GPIO_ACTIVE_HIGH>;
++                         linux,default-trigger = "wf188:green:power";
++ 			default-state = "off";
++Index: linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce/arch/arm64/boot/dts/qcom/qcom-ipq6018-cig-wf188n.dts
++===================================================================
++--- linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce.orig/arch/arm64/boot/dts/qcom/qcom-ipq6018-cig-wf188n.dts
+++++ linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce/arch/arm64/boot/dts/qcom/qcom-ipq6018-cig-wf188n.dts
++@@ -309,27 +309,27 @@
++ 		pinctrl-names = "default";
++ 
++ 		led@25 {
++-			label = "led_5g";
+++			label = "wf188n:green:wifi5g";
++ 			gpios = <&tlmm 25 GPIO_ACTIVE_HIGH>;
++-			linux,default-trigger = "wf188:green:5g";
+++			linux,default-trigger = "wf188n:green:wifi5g";
++ 			default-state = "off";
++ 		};
++ 		led@24 {
++-			label = "led_2g";
+++			label = "wf188n:green:wifi2g";
++ 			gpios = <&tlmm 24 GPIO_ACTIVE_HIGH>;
++-			linux,default-trigger = "wf188:green:2g";
+++			linux,default-trigger = "wf188n:green:wifi2g";
++ 			default-state = "off";
++ 		};
++ 		led@18 {
++-			label = "led_eth";
+++			label = "wf188n:green:eth";
++ 			gpios = <&tlmm 18 GPIO_ACTIVE_HIGH>;
++-			linux,default-trigger = "wf188:green:eth";
+++			linux,default-trigger = "wf188n:green:eth";
++ 			default-state = "off";
++ 		};
++                 led_power: led@16 {
++-                        label = "led_pwr";
+++                        label = "wf188n:green:power";
++                         gpios = <&tlmm 16 GPIO_ACTIVE_HIGH>;
++-                        linux,default-trigger = "wf188:green:power";
+++                        linux,default-trigger = "wf188n:green:power";
++ 			default-state = "off";
++ 		};
++ 	};
++Index: linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce/arch/arm64/boot/dts/qcom/qcom-ipq6018-edgecore-eap101.dts
++===================================================================
++--- linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce.orig/arch/arm64/boot/dts/qcom/qcom-ipq6018-edgecore-eap101.dts
+++++ linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce/arch/arm64/boot/dts/qcom/qcom-ipq6018-edgecore-eap101.dts
++@@ -337,21 +337,21 @@
++ 		pinctrl-names = "default";
++ 
++ 		led@25 {
++-			label = "wifi5g";
+++			label = "eap101:green:wifi5g";
++ 			gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>;
```

Comment:
John's change earlier today switched the EAP101 LEDs from ACTIVE_HIGH to ACTIVE_LOW.
This may not merge as a result.

Also, we need to be sure that the LEDs are ON/OFF when expected.  If ACTIVE_HIGH and ACTIVE _LOW are wrong, the LEDs may be on when asked to be off and vice versa.

Be sure to test all the targets.

Reply:
Sure, I'll do that. Some how I didn't see John Crispin changes in my kernel board specific files, even though I'm up to date with latest changes. I tested with one WIFI5 AP and one WIFI6 AP currently, I'll test on all APs in the lab tomorrow.

Reply:
Nag checked and validated.